docs/docs/copilot.md

```diff
@@ -0,0 +1,84 @@
+---
+id: tooljet-copilot
+title: Copilot
+---
+
+:::info BETA
+ToolJet Copilot is currently available in public beta for all the users.
+:::
+
+**ToolJet Copilot** helps you write your queries faster. It uses OpenAI to suggest queries based on your data. 
+
+## Activating Copilot
+
+To access the Copilot, each user, including administrators, can sign up for the waitlist program individually. After signing up, each user will receive a unique API key associated with their account. **Only administrators**, however, will have the privilege to enable Copilot within the workspace settings. Administrators can navigate to the Copilot section in the workspace settings and toggle on the "Enable Copilot" option using their respective API key.
+
+You can join the waitlist here: **https://tooljet.com/copilot**
+
+:::info
+- Copilot can be used by users who have **permissions to edit** the app with the copilot setup.
+:::
+
+<div style={{textAlign: 'center'}}>
+
+<img className="screenshot-full" src="/img/copilot/enable.png" alt="ToolJet Copilot" />
+
+</div>
+
+## Using Copilot
+
+Once activated, the Copilot feature can be accessed while editing any query within the transformations section in the query editor.
+
+For Copilot to operate, it requires a plain English prompt. Copilot possesses the ability to interact with and access components, along with generating code for specific actions related to those components. This capability is made possible by Copilot's awareness of the application's current state.
+
+### Example: Generating a SQL statement 
+
+We will demonstrate the usage of copilot by generating a SQL query to fetch the order details from a PostgreSQL database based on the key provided in the REST API data.
+
+- Let's create a RESTAPI query using this mock api endpoint: https://fakestoreapi.com/users/1
+- Enable transformation on the query editor
+
```

Comment:
Let's try to avoid statements starting with "Let us", "We will" in the documentation, please stick to a constant voice.